Tests/SwiftParserTest/TypeTests.swift

```diff
@@ -170,3 +170,6 @@ final class TypeTests: XCTestCase {
     )
   }
 }
+
+import SwiftBasicFormat
+import _SwiftSyntaxTestSupport
```

Comment:
👀 Why do we need this?

Reply:
Haha, thanks. That was from when I was making the few BasicFormat issues :D